modules/magniteAnalyticsAdapter.js

```diff
@@ -939,6 +952,61 @@ magniteAdapter.track = ({ eventType, args }) => {
   }
 };
 
+const handleNonBidEvent = function(args) {
+  const {seatnonbid, auctionId} = args;
+  const auction = deepAccess(cache, `auctions.${auctionId}.auction`);
+  const adUnits = auction.adUnits;
```

Comment:
WE should probably safe check in case the auction was not found for some reason:

```
  const auction = deepAccess(cache, `auctions.${auctionId}.auction`) ;
  // if no auction just bail
  if (!auction) logWarn(`Unable to match nonbid to auction`) && return;
```